onnxruntime/core/graph/model.h

```diff
@@ -46,7 +46,7 @@ class Model {
 
   // NOTE: after calling this constructor, <*this> model will
```

Comment:
comment is still true. The caller relinquishes the instance of ModelProto.

Reply:
This changes the interface and the calling site. People will have to change the code both to invoke and to consider the impact on the logic.

---
In reply to: [385381979](https://github.com/microsoft/onnxruntime/pull/3103#discussion_r385381979) [](ancestors = 385381979)

Reply:
Internal only header though so should only impact our code.

---
In reply to: [385383519](https://github.com/microsoft/onnxruntime/pull/3103#discussion_r385383519) [](ancestors = 385383519,385381979)